setup/README.md

```diff
@@ -0,0 +1,87 @@
+# Orka VM Setup Script Usage
+
+## This script automates the configuration and setup of macOS Orka VMs for optimal performance. It configures system settings, installs required tools, and prepares the VM for use with Orka
+
+### System requirements
+
+- macOS IPSW file (Tahoe, Sequoia, or Sonoma)
+- Admin/sudo privileges on the VM
+- Internet connection for downloading packages
+- Terminal access (via SSH, Screen Sharing, or direct console)
+
+### Before running
+
+- Ensure you have administrator access to the VM
+- Close any important applications (they will be terminated during cleanup)
+- Save any work in progress (the VM will reboot automatically)
+
+### Installation
+
+#### Download via GitHub
+
+- `bash /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/macstadium/orka-images/setup/setup.sh)"`
+
+### Installation steps when running the script
+
+#### On MacOS Sequoia
+
+- When prompted for Terminal permissions access, click 'Accept'
+- Terminal will need full disk access to run the script. Configure this by going to your System Settings -> Privacy and Security -> Full Disk Access and adding 'Terminal' by clicking the + button and searching for the app in the search bar.
+- Enter your password when prompted during the script installation process
+- When the script shell displays `sh-3.2#` type `exit` and press `Enter` to continue script installation
+- Upon system reboot, re-open the Terminal application and enter `sudo launchctl list sysctl` to confirm the Orka sys-daemon script has installed. You should see an output similar to:
+
+```markdown
```

Comment:
Somewhat `nit` - but this still renders a bit awkwardly with the trailing `);`

<img width="545" height="290" alt="Screenshot 2025-10-02 at 2 33 12 PM" src="https://github.com/user-attachments/assets/6532ce0d-271a-4fbd-946b-83632dfc4586" />

Reply:
no worries, pushed a new commit